Mathlib/Analysis/SpecialFunctions/Exp.lean

```diff
@@ -416,6 +416,11 @@ theorem isTheta_exp_comp_one {f : α → ℝ} :
 set_option linter.uppercaseLean3 false in
 #align real.is_Theta_exp_comp_one Real.isTheta_exp_comp_one
 
+lemma summable_exp_neg_nat : Summable (fun n : ℕ ↦ exp (-n)) := by
```

Comment:
Maybe generalize this to `exp (-n * x)` with positive `x`? I'd expect this to be more useful than the special case `x = 1`.

Reply:
Good point, I added a lemma showing that `Summable (fun n : ℕ ↦ exp (n * a)) ↔ a < 0` and deduced the case at hand from this.